.github/PULL_REQUEST_TEMPLATE/new_ea_pr_template.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+`<description>`
+
+## Steps to Test
+
+1. `yarn && yarn setup`
```

Comment:
Should we have one section for automated tests and one for manual acceptance (if applicable)?

Reply:
Do you mean `automated` as unit + integration tests, and `manual` as sanity testing?

Reply:
I think I'll leave it for now since IMO it may confuse external contributors to distinguish the different levels of tests like this.